packages/forms/src/Concerns/HasStateBindingModifiers.php

```diff
@@ -29,11 +29,17 @@ public function stateBindingModifiers(array $modifiers): static
         return $this;
     }
 
-    public function applyStateBindingModifiers($expression): string
+    public function applyStateBindingModifiers($expression, ?string $except = null): string
     {
         $modifiers = $this->getStateBindingModifiers();
 
-        return implode('.', array_merge([$expression], $modifiers));
+        $merged = array_merge([$expression], $modifiers);
```

Comment:
Logically, it makes more sense to filter the array to remove the modifiers, then merge the expression

We can probably do it with a collection, chaining `->implode()` at the end

Reply:
I changed it, see comment below.